config/locales/en.yml

```
@@ -1801,10 +1801,17 @@ user_mailer.account_reset_request.cancel: Don’t want to delete your account? S
 user_mailer.account_reset_request.header: Your account will be deleted in %{interval}
 user_mailer.account_reset_request.intro_html: 'As a security measure, %{app_name} requires a two-step process to delete your account:<br><br> Step One: There is a waiting period of %{waiting_period} if you have lost access to your authentication methods and need to delete your account. If you locate your authentication methods, you can sign in to your %{app_name} account to cancel this request.<br><br> Step Two: After the waiting period of %{waiting_period}, you will receive an email that will ask you to confirm the deletion of your %{app_name} account. Your account will not be deleted until you confirm.'
 user_mailer.account_reset_request.subject: How to delete your %{app_name} account
+user_mailer.account_verified_but_not_connected.change_password_link: change your password
+user_mailer.account_verified_but_not_connected.contact_link: contact us
+user_mailer.account_verified_but_not_connected.help_html: If you did not perform this action, please %{contact_link_html} and sign in to %{change_password_link_html}.
+user_mailer.account_verified_but_not_connected.instructions_html: Sign back in at the <strong>%{sp_name}</strong> website to connect your verified information and access services.
```

my vote is to move the strong tag into the ERB
```suggestion
user_mailer.account_verified_but_not_connected.instructions_html: Sign back in at the %{sp_name_html} website to connect your verified information and access services.
```